server/jamovi/server/i18n.py

```diff
@@ -0,0 +1,56 @@
+
+import os.path
+import json
+
+from .utils import conf
+
+
+class I18n:
+    def __init__(self, session):
+        self._session = session
+        self.i18n_def = None
+        self._loaded_code = None
+
+    @property
+    def locale(self):
+        return self._loaded_code
```

Comment:
strictly speaking, locale isn't exactly the same as language code:

https://en.wikipedia.org/wiki/Locale_(computer_software)